githook/prepare-commit-msg

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+
+import sys
+
+def is_integer(value: str) -> bool:
+    try:
+        int(value)
+        return True
+    except ValueError as _:
+        pass
+
+    return False
+
+def check_commit_message() -> int:
+    commit_message_filename: str = sys.argv[1]
+    with open(commit_message_filename) as file:
+        first_line: str = file.readline().strip()
+
+    issue_name = first_line.split(" ")[0]
+
+    # Allow fixup commits
+    if issue_name == "fixup!":
+        return 0
+
+    if issue_name == "WIP":
+        return 0
+
+    if issue_name.startswith("KTOR-") and is_integer(issue_name[5:]):
+        return 0
+
+    print(f"Commit name should starts with issue number(like KTOR-1234). Current prefix is '{issue_name}'")
```

Comment:
should start_

Reply:
thanks, will fix

Reply:
Just a catch: we could use a real ktor ticket describing commit message policy instead of 1234